lib/sinon/spy.js

```diff
@@ -8,10 +8,15 @@
   */
 "use strict";
 
-require("./extend");
-require("./call");
-require("./format");
+var extend = require("./extend");
+var deepEqual = require("./util/core/deep-equal");
+var functionName = require("./util/core/function-name");
+var functionToString = require("./util/core/function-to-string");
+var getPropertyDescriptor = require("./util/core/get-property-descriptor");
 var sinon = require("./util/core");
```

Comment:
I have to use `sinon.format` from core instead of`var format = require('.../format')`.